### DISCUSSION TITLE
Splitting .ino file to multiple "modules"

### DISCUSSION THREAD

**Discussion by dukess** [fredlcore/BSB-LAN#394] [General]
I tried to move ~100 KB of code to some .h files. May this can improve readability. It is interesting for us?
https://github.com/dukess/BSB-LAN/tree/splitted

**Comment by dukess** [comment]
@fredlcore Just call you

**Comment by fredlcore** [comment]
Thanks! I know it's better to split large portions of code into smaller pieces (@thorbennehmer suggested that as well a while ago), but so far I was always in favor of having all code in one place because it makes it easier to jump from one occurrence to another.
But I had a look at your approach and I kind of like it because it's still quite intuitive, so if you have given it some thorogh testing and it works well, then feel free to send a PR!

**Comment by dukess** [comment]
Ok!
I think it possible do code rewriting for correct split to modules in future. I don't want do it right now because it require more RAM which not enough on Mega.

**Reply by fredlcore** [reply]
Ok, just let me know when you plan to do it, so I can stop working on the code for a while.
